lib/common/pacemakerd_client.c

```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright 2004-2019 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
+
+#include <crm_internal.h>
+#include <stdlib.h>
+#include <time.h>
+
+#include <crm/msg_xml.h>
+#include <crm/common/mainloop.h>
+#include <crm/common/pacemakerd_types.h>
+
+CRM_TRACE_INIT_DATA(pacemakerd);
```

Comment:
Hadn't checked what was behind it but as I found it multiple times in other libraries I was assuming it added some static stuff as well.
But to be honest I don't see where it currently would be defined to anything else than NIL.

Reply:
I didn't look that far, I wouldn't be surprised if it was unnecessary. :) The only library that has it twice is an accident because two libraries were merged into one.